apps/web/app/(use-page-wrapper)/auth/saml-idp/page.tsx

```diff
@@ -0,0 +1,18 @@
+import type { PageProps } from "app/_types";
+import { notFound } from "next/navigation";
+import { z } from "zod";
+
+import SamlIdpClient from "~/auth/saml-idp/saml-idp-view";
+
+const querySchema = z.object({
+  code: z.string(),
+});
+
+export default async function SamlIdpPage({ params }: PageProps) {
+  const parsed = querySchema.safeParse(params);
```

Comment:
you mean `safeParse(searchParams)`?

Reply:
wow, thanks for catching this, addressed!